wp-content/themes/access/src/js/modules/screener.js

```diff
@@ -152,9 +152,39 @@ class Screener {
       this._goToStep(this._$steps[0]);
     }
 
+    this._scenarioAnalysis();
```

Comment:
Move this function into the Screener module, since it is specific to the Screener